CHANGELOG.md

```diff
@@ -2,6 +2,69 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x will End of Life on September 8th, 2021.** Check [this guide](https://www.twilio.com/docs/video/migrating-1x-2x) to plan your migration to the latest 2.x version. Support for the 1.x version ended on December 4th, 2020.
 
+2.17.0 (In Progress)
+====================
+**Preflight API Pilot**
+-----------------------
+
+- This release adds a new experimental API `runPreflight` to help test the connectivity with twilio servers.
```

Comment:
We should mention that this is an update of the old preflight API and it's not a new concept.

Reply:
yeah, I think this need some rewording and update, but that we can do it before going GA